app/models/TaxIdentificationNumber.scala

```diff
@@ -22,6 +22,7 @@ sealed trait TaxIdentificationNumber {
   def value: String
 }
 
+//TODO: add TRN
```

Comment:
Addressing this in a later ticket?

Reply:
separate because it'll be its own refactor - i don't think it's covered in any future ticket so we'll probably do it as part of techdebt - hence the todo to remind us!